documentation/BrowserChannel.md

```diff
@@ -24,39 +24,45 @@ Also, this component create notifications for every connected client at any time
 This small and documented client illustrates the workflow.
 
 ```javascript
-var io = require("socket.io-client");
+var io = require("ws");
```

Comment:
Thanks - what a blunder! I've checked in a working sample with instructions now at examples/browserChannelClient